src/Illuminate/Support/Str.php

```diff
@@ -521,6 +521,18 @@ public static function ucfirst($string)
         return static::upper(static::substr($string, 0, 1)).static::substr($string, 1);
     }
 
+    /**
+     * Check if two strings are equal, ignoring case.
+     *
+     * @param  string  $a
+     * @param  string  $b
+     * @return bool
+     */
+    public static function equalsIgnoreCase($a, $b)
+    {
+        return static::lower($a) === static::lower($b);
```

Comment:
We can use the native `strcasecmp` function and save these two static calls IMHO.

Reply:
Actually one can just use `strcasecmp` directly, no? The wrapper would be too thin to be much useful.

Reply:
Scratch that. I’ve just realized `strcasecmp` is single-byte. What a bummer.